.grabl/automation.yml

```diff
@@ -113,175 +113,83 @@ build:
         tail -f -n +1 ./dist/typedb-all-linux/server/logs/typedb.log
     test-performance-typedb-core-benchmark:
       machine: 16-core-32-gb
-      image: vaticle-ubuntu-21.04
+      image: vaticle-ubuntu-20.04
       dependencies: [test-performance-typedb-core-server]
       timeout: "30m"
       command: |
-       bazel run //:benchmark -- \
-         --database typedb \
-         --address $GRABL_EXPORT_PERFORMANCE_TYPEDB_CORE_URI \
-         --config /home/grabl/$GRABL_REPO/config/simulation.yml
-    test-performance-typedb-cluster-one-server:
+        bazel run //:benchmark -- \
+          --database typedb \
+          --address $GRABL_EXPORT_PERFORMANCE_TYPEDB_CORE_URI \
+          --config /home/grabl/$GRABL_REPO/config/simulation.yml \
+#          --factory $GRABL_TRACING_URI \
+#          --org $GRABL_OWNER \
+#          --repo $GRABL_REPO \
+#          --commit $GRABL_COMMIT \
+#          --username $GRABL_OWNER \
+#          --token $GRABL_TOKEN
```

Comment:
factory-tracing is currently not working due to SSL issues, so we will leave this commented